core/sources/domains.move

```diff
@@ -135,7 +149,19 @@ module aptos_names::domains {
         token_helper::initialize(account);
     }
 
-    fun register_domain_generic(sign: &signer, domain_name: String, num_years: u8) acquires NameRegistryV1, RegisterNameEventsV1, SetNameAddressEventsV1 {
+    public fun init_reverse_lookup_registry_v1(account: &signer) {
+        assert!(signer::address_of(account) == @aptos_names, ENOT_AUTHORIZED);
```

Comment:
nit: Wrap ENOT_AUTHORIZED to be consistent with other error codes.

Reply:
Done